### DISCUSSION TITLE
Get a list of all active streams/stream keys

### DISCUSSION THREAD

**Discussion by domainzero** [OvenMediaLabs/OvenMediaEngine#1488] [Help] [answered]
Is there an efficient way to get a list of all the currently active streams/stream keys?
Trying to build a status page that has a list of active streams.

**Comment by bchah** [comment, marked as answer]
The REST API supports this: https://airensoft.gitbook.io/ovenmediaengine/rest-api/v1/virtualhost/application/stream

**Reply by domainzero** [reply]
I can't read! Thanks for the pointer.
